src/components/Admin.tsx

```diff
@@ -26,41 +30,72 @@ const Admin = () => {
             <li>
               <Button
                 appearance="transparent"
-                onClick={() =>
-                  seturl(
-                    _spPageContextInfo.webAbsoluteUrl +
-                      "/Lists/Disclaimers/AllItems.aspx"
-                  )
-                }
+                onClick={() => seturl(globalList + "Disclaimers/AllItems.aspx")}
               >
                 Disclaimer Statements (Global)
               </Button>
             </li>
-            <li>Department of Defense Activity Address Directory (DoDAAD)</li>
+            <li>
+              <Button
+                appearance="transparent"
+                onClick={() => seturl(globalList + "DODAAC/AllItems.aspx")}
+              >
+                DODAACs
+              </Button>
+            </li>
           </ul>
         </>
       )}
       {program && programPerms.data && (
         <>
           <Title3>{program} Program Office Items</Title3>
           <ul>
-            <li>Contractors</li>
-            <li>Contracts</li>
             <li>
               <Button
                 appearance="transparent"
                 onClick={() =>
-                  seturl(
-                    _spPageContextInfo.webAbsoluteUrl +
-                      "/" +
-                      program +
-                      "/Lists/Disclaimers/AllItems.aspx"
-                  )
+                  seturl(programList + "Contractors/AllItems.aspx")
+                }
+              >
+                Contractors
```

Comment:
Do we want to turn on any settings in the list to help maintain valid relationships? The one I am thinking, is restrict them from removing a Contractor if there are any Contractor POCs which have it set as the Contractor they work for.

Reply:
Per our discussion the other day, this isin't something worth implementing